Makefile

```diff
@@ -140,6 +140,10 @@ bash: compose_build
 refresh: kill compose_build
 	$(COMPOSE) run -e SKIP_DUMMY=$$SKIP_DUMMY app bash -c '$(WAIT_FOR_DB) $(PYTHON_MIGRATE)&&$(LOAD_LOCALES)&&$(LOAD_COUNTRIES)&&$(LOAD_DUMMY_EXPERIMENTS)'
 
+dependabot_approve:
+	echo "Install and configure the Github CLI https://github.com/cli/cli"
+	gh pr list | grep "chore(deps" | awk '{print $$1}' | xargs -n1 gh pr review -a -b "@dependabot squash and merge"
```

Comment:
Hmm, doesn't appear this is an [option](https://cli.github.com/manual/gh_pr_list#options)*, but it would be nice if were filtering by PRs that are passing CI, so we're _trying_ to merge failing PRs (I know it wouldn't allow us anyway, but still).

* Unless we make CircleCI apply a `passing` label to PRs that are passing, then we could filtering by label, but feels like more effort than it's worth

Reply:
They can't merge if they don't pass so even if they're approved nothing will happen.

Reply:
And this step can be done before they even run in CI.  And they'll often rebase themselves as others merge and re run CI so, I don't think worrying about whether they're passing when this is run matters.

Reply:
Yup. I mostly suggested this as a nice-to-have since someone not familiar with this process could go to the PR page and get confused seeing `@dependabot squash and merge` comments on failing PRs. Ignore me.